code/modules/uplink/uplink_items/job.dm

```diff
@@ -369,3 +369,14 @@
 	restricted_roles = list(JOB_ASSISTANT, JOB_MIME, JOB_CLOWN)
 	restricted = TRUE
 	refundable = FALSE
+
+
+/datum/uplink_item/role_restricted/reticence
+	name = "Reticence Cloaked Assasination exosuit"
+	desc = "A silent, fast, and nigh-invisible but exepctionally fragile miming exosuit! \
```

Comment:
It's spelled as "exceptional", also I don't believe you need to capitalize "Near-Silenced"

Reply:
fixed